audio/stereo_panner_node.rs

```diff
@@ -0,0 +1,137 @@
+use block::{Block, Chunk, Tick, FRAMES_PER_BLOCK};
+use euclid::Vector3D;
```

Comment:
Please name this file `stereo_panner.rs`. Also, for this to work you'll need a `mod stereo_panner` in `audio/lib.rs`